client/components/like-button/index.jsx

```diff
@@ -1,7 +1,8 @@
 /**
  * External Dependencies
  */
-var React = require( 'react/addons' ),
```

Comment:
>Did `react/addons` contain react as well? 

Yes, `react/addons` is essentially a superset of React which adds the `addons` property methods.